src/main/kotlin/com/project/twiliospring/config/TwilioConfiguration.kt

```diff
@@ -16,4 +19,11 @@ class BeanConfiguration(
     fun initTwilio() {
         Twilio.init(twilioSID, twilioToken)
```

Comment:
can be a part of Bean creation

Reply:
It's initialization of twilio library, not a part of just one bean